### PR TITLE
[11.0][MIG] product_weight

### PR DESCRIPTION
**Pull request by ProcessControl-JHF**
cc @guadaltech

### CODE REVIEW

File: product_weight/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Usage
+=====
+
+You can update the weight of one product clicking on the button under 
+the weights. 
+Or you can select the products you want to update from the tree list.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/135/10.0
```
Comment (bodedra) [inline product_weight/README.rst]: change runbot 10.0 to 11.0 or use new structure of README

File: product_weight/wizard/product_weight_update_view.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	      <footer>
+	        <button name="update_single_weight" string="_Apply" type="object" class="oe_highlight"/>
+	        or
+	        <button string="Cancel" class="oe_link" special="cancel" />
+	      </footer>
+	    </form>
+      </field>
+    </record>
+
+
```
Comment (bodedra) [inline product_weight/wizard/product_weight_update_view.xml]: Remove empty line

File: product_weight/wizard/product_weight_update_view.xml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      <field name="name">Update Product Weight</field>
+      <field name="type">ir.actions.act_window</field>
+      <field name="res_model">product.weight.update</field>
+      <field name="view_type">form</field>
+      <field name="view_mode">form</field>
+      <field name="view_id" ref="view_product_weight_update"/>
+      <field name="target">new</field>
+    </record>
+
+
```
Comment (bodedra) [inline product_weight/wizard/product_weight_update_view.xml]: Same. Remove empty line

File: product_weight/wizard/product_weight_update_view.xml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+                key2="client_action_multi"
+                res_model="product.weight.update"
+                src_model="product.template"
+                view_mode="form"
+                target="new"
+                view_type="form"
+                view_id="view_product_weight_update_multi" />
+    
+</odoo>
+
```
Comment (bodedra) [inline product_weight/wizard/product_weight_update_view.xml]: Same. Remove empty line

File: product_weight/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2013 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# Copyright (C) 2015 Akretion (<http://www.akretion.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Product Weight Calculation',
+    'version': '11.0.1.0.0',
+    'author': 'Savoir-faire Linux,Akretion,Odoo Community Association (OCA)',
+    'website': 'http://www.savoirfairelinux.com',
```
Comment (bodedra) [inline product_weight/__manifest__.py]: website should be `https://github.com/OCA/product-attribute`

**Comment by pedrobaeza** [comment]
@jhumfer good to see you around and I hope this is the first of lot of contributions :smiley: (and also from your company). Let's see the result of the CI for approving this.

**Review by pedrobaeza — APPROVED** [review]

**Review by rvalyi — APPROVED** [review]
thanks for the contrib!

**Comment by pedrobaeza** [comment]
I have simplified commit history. @bodedra do you give your final blessing for merging this?

**Review by bodedra — APPROVED** [review]
LGTM :+1:

**Pull request merged**